### PR TITLE
Update b2b-extranet.md

### PR DESCRIPTION
**Pull request by msbemba**
Added licensing requirements

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue #https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2782

File: microsoft-365/solutions/b2b-extranet.md [hunk 1]
```diff
@@ -27,6 +27,12 @@ This article walks through the steps to create a package of resources (in this c
 
 Before you begin, create the site or team that you want to share with the partner organization and enable it for guest sharing. See [Collaborate with guests in a site](collaborate-in-site.md) or [Collaborate with guests in a team](collaborate-as-team.md) for more information. We also recommend that you review [Create a secure guest sharing environment](create-secure-guest-sharing-environment.md) for information about security and compliance features that you can use to help maintain your governance policies when collaborating with guests.
 
+## License requirements
+
+[!INCLUDE [Azure AD Premium P2 license](../../../includes/active-directory-p2-license.md)]
+
+Specialized clouds, such as Azure Germany, and Azure China 21Vianet, are not currently available for use.
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/b2b-extranet.md]:
```suggestion
Specialized cloud services, such as Azure Germany and Azure China, are not currently available for use.
```

Comment (JohanFreelancer9) [reply microsoft-365/solutions/b2b-extranet.md]: My understanding is that "Azure China" is in fact "Azure operated by 21Vianet". If I'm wrong please ignore the edit.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: microsoft-365/solutions/b2b-extranet.md (hunk 1 above)
Comment (msbemba) [reply microsoft-365/solutions/b2b-extranet.md]: Thank you @JohanFreelancer9. I'll ignore the edit as the doc needs to have Azure China 21Vianet as per the standard.

Comment (JohanFreelancer9) [inline microsoft-365/solutions/b2b-extranet.md]:
```suggestion
Specialized clouds, such as Azure Germany and Azure China 21Vianet, are not currently available for use.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @msbemba I have noted your comment, please just remove the comma as indicated

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MikePlumleyMSFT Please merge this PR. Thnaks.

**Comment by MikePlumleyMSFT** [comment]
@msbemba the file you're linking to does not appear to be in this repo.

**Comment by msbemba** [comment]
Thank you @MikePlumleyMSFT. I have removed the link now with just the license information.

**Pull request merged**